src/databaseAdapter.js

```diff
@@ -51,10 +51,60 @@ const getAllProjects = async () => {
   return docs.map((doc) => doc.data());
 };
 
+//Fetch a project ID
+const getProjectIdByName = async (projectName) => {
+  let { docs } = await db
+    .collection(projectsCollection)
+    .where('projectName', '==', projectName)
+    .get();
+
+  //Will select the last (only) document with that name
+  return docs.map((doc) => doc.id);
```

Comment:
returning an array doesn't make much sense when we're searching for one value. This could be something like
`return docs[0]?.id`
That should safely return the id of the first and ideally only document and return undefined without throwing an error if one isn't found